packs/nest/actions/fan.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: fan
+runner_type: run-python
+description: Manage fan state
+enabled: true
+entry_point: fan.py
+parameters:
+  state:
+    type: string
+    description: State to set the fan
+    enum:
+      - "on"
```

Comment:
Stylistically, not sure how best to approach. TIL that `on` in YAML is actually a `true` value, and casts as such when parsed. So, `"on"` for sure to keep the literal string, but sort of not necessary with "auto".

Certainly open to suggestions here.